lambda.tf

```diff
@@ -1,10 +1,11 @@
 module "lambda" {
-  source = "github.com/pbs/terraform-aws-lambda-module?ref=0.3.0"
+  source = "github.com/pbs/terraform-aws-lambda-module?ref=993a9219ba5798bc1c007c44d7559cadd171e1a5"
```

Comment:
TODO: Update this once https://github.com/pbs/terraform-aws-lambda-module/pull/11 gets merged.